cfiles/tests/outputs/test_script.sh_output.txt

```diff
@@ -0,0 +1,3 @@
+rm -f *.o node-test edge-test edgelist-test nodelist-test graph-test
+gcc -g -Wall -pthread -o node-test node-test.c
+gcc -g -Wall -pthread -o edge-test edge-test.c
```

Comment:
this is an error because it thought ./test_script.sh was an executable, so it created an output file but it should be deleted

Reply:
yup deleted it -- thanks for catching